src/Language/AST/NamedTypeNode.php

```diff
@@ -9,6 +9,6 @@ class NamedTypeNode extends Node implements TypeNode
     /** @var string */
     public $kind = NodeKind::NAMED_TYPE;
 
-    /** @var NameNode */
+    /** @var string */
```

Comment:
Can you check if this is correct? There are some new errors in baseline I guess should not be there like

```
"#^Cannot access property \\$value on string\\.$#"
src/Validator/Rules/ProvidedRequiredArgumentsOnDirectives.php
```

Reply:
You're right (it can be either NameNode or string). Fixed in a702169